rest/client/rest.go

```diff
@@ -734,3 +735,46 @@ func (c *communicatorImpl) SendNotification(ctx context.Context, notificationTyp
 
 	return nil
 }
+
+func (c *communicatorImpl) GetDockerLogs(ctx context.Context, evergreenID string, startTime time.Time, endTime time.Time) (*cloud.LogInfo, error) {
+	path := fmt.Sprintf("/host/%s/logs", evergreenID)
+	if startTime != util.ZeroTime && endTime != util.ZeroTime {
```

Comment:
We're unfortunately inconsistent with zero time in our data models, sometimes using Go's zero time and sometimes using the Unix zero time. Because of this you can use util.IsZeroTime to check both cases